package/network/utils/iproute2/Makefile

```diff
@@ -232,11 +225,6 @@ define Package/tc-full/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tc/tc $(1)/usr/libexec/tc-full
 endef
 
-define Package/tc-mod-iptables/install
-	$(INSTALL_DIR) $(1)/usr/lib/tc
-	$(CP) $(PKG_BUILD_DIR)/tc/m_xt.so $(1)/usr/lib/tc
-endef
```

Comment:
nice, the shared libs were removed here: https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=45dc10463dd5a51eef944b110a4f046feb1b46db